languages/scala/reference/exercise-concepts/space-age.md

```diff
@@ -0,0 +1,17 @@
+# Concepts of space-age
+
+[Example implementation](https://github.com/exercism/scala/blob/master/exercises/space-age/example.scala)
+
+## General
+
+- object or case class: used as class wrapper for exercises
+- methods: used as main entry point for the exercise and helper methods
+- functions: each method is a function
```

Comment:
Maybe talk about `partial function` since 
```
  def onMercury(seconds: Double): Double = toPlanetAge(daysToMercuryAge, seconds)
```

are basically partial function:
```
  val onMercury: Double => Double = toPlanetAge(daysToMercuryAge, _)
```

Reply:
Excellent point!

Reply:
I've updated the PR.